changelog/unreleased/reconfiguration-completion-detection.yml

```diff
@@ -0,0 +1,3 @@
+message: Provide mechanism to detect completion of reconfiguration on the proxy path
```

Comment:
"Provide mechanism to detect completion of reconfiguration on the proxy path. This is for internal testing only."